src/content/configuration/dev-server.md

```diff
@@ -615,17 +653,17 @@ module.exports = {
 };
 ```
 
-Usage via the CLI
+Usage via the CLI:
 
 ```bash
 npx webpack serve --history-api-fallback
 ```
 
 For more options and information, see the [connect-history-api-fallback](https://github.com/bripkens/connect-history-api-fallback) documentation.
 
-## `devServer.host`
+## devServer.host
 
-`string = 'localhost'`
+`string`
```

Comment:
We have `local-ip`/`local-ipv4`/`local-ipv6`, make sense to mark, it I will fix schema in dev server

Reply:
I just noticed it too, need to update the schema. I can send a PR.

Reply:
Already WIP on this :+1: Also will check other values